### DISCUSSION TITLE
FastConformer-Longformer

### DISCUSSION THREAD

**Discussion by GrigorKirakosyan** [NVIDIA-NeMo/NeMo#9991] [Q&A]
Hi NeMo team,
Do you plan to release En Large FastConformer-Long-CTC-BPE model trained with Local Attention and Global token?

**Comment by nithinraok** [comment]
we released 1B model but not the large model here: https://huggingface.co/spaces/nvidia/parakeet-tdt_ctc-1.1b

**Comment by GrigorKirakosyan** [comment]
Thanks, I have seen this. I was interested to know about Large-FastConformer(~120M) model trained with this config:
https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/fastconformer/long_fastconformer/fast-conformer-long_ctc_bpe.yaml
